server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/ResponseSerializer.scala

```diff
@@ -225,6 +225,13 @@ object ResponseSerializer {
     case error: WebSocketError => webSocketErrorWrites.writes(error)
   }
 
+  private implicit val typeNameReads: Reads[TypeName] = {
+    case JsString(value) if value.equals(MailboxTypeName.asString()) => JsSuccess(MailboxTypeName)
```

Comment:
`case JsString(MailboxTypeName.asString) => xxx` is more straightforward?

Reply:
does not compile i think